libs/natparse/src/test/java/org/amshove/natparse/parsing/DefineDataParserShould.java

```diff
@@ -521,6 +521,17 @@ void parseAnArrayWithWhitespaceBeforeTheSlash()
 			""");
 	}
 
+	/*	@Test TODO: make this work
```

Comment:
You can comment this in and add a `@Disabled` annotation, this will skip this test.

e.g.
`@Disabled("Does not work yet")`

Also change the `TODO` to `TODO(lexermode)` so that I can find it when I'm implementing this

Reply:
Build fails when doing this, Google seems to confirm it's named @Disabled. What am I missing?

C:\java_devenv\git\natls-tryg\libs\natparse\src\test\java\org\amshove\natparse\parsing\DefineDataParserShould.java:524: error: cannot find symbol
        @Disabled
         ^
  symbol:   class Disabled
  location: class DefineDataParserShould
1 error

FAILURE: Build failed with an exception.

Reply:
You`re missing the import then:
`import org.junit.jupiter.api.Disabled;`